code/Peer.md

```diff
@@ -0,0 +1,173 @@
+# Peer
+
+Peer is the top-level entity in Storj. In the production network there's every peer runs in a standalone process. For tests multiple peers may run in a single process.
+
+Peers describe how a particular network participant is setup, wired together and run.
+
+Currently there network contains peers:
+
+1. Bootstrap
+2. Storage Node
+3. Satellite
+4. Uplink
+5. Version Control
```

Comment:
version control peer? what does this mean?

Reply:
Its our newest addition to ensure the network is up to date.
I was part of my versioning PR

Reply:
very interesting